src/components/Tabs/TabsComponent.tsx

```diff
@@ -5,18 +5,27 @@ import {
   Tab,
   TabPanel,
   ChakraProvider,
+  Flex,
 } from "@chakra-ui/react";
 import chakraTheme from "@/styles/chakraTheme";
 import { populatedGameWithId } from "@/server/db/models/GameModel";
+import DeleteVideoTrailer from "../GameScreen/DeleteVideoTrailerComponent";
+import AddOrEditVideoTrailer from "../GameScreen/AddVideoTrailerComponent";
 import { ChangeEvent, Dispatch, useState } from "react";
-
+import ReactPlayer from "react-player/lazy";
 interface Props {
   mode: string;
   gameData: populatedGameWithId;
   setGameData?: Dispatch<populatedGameWithId>;
+  admin?: boolean;
```

Comment:
can you change the name of this variable so it describes its purpose better (since its not just administrators)? "authorized" maybe?